<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRoundedRectItemDialog</class>
 <widget class="QDialog" name="QtRoundedRectItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRoundedRectItemDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="box_radius_x">
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_radius_y">
     <property name="text">
      <string>Radius Y</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="box_height">
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>128.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_contour_pen_width">
     <property name="text">
      <string>Contour pen width</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_x">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="box_radius_y">
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_radius_x">
     <property name="text">
      <string>Radius X</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="box_width">
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>64.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_width">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_y">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="box_y">
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_height">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="box_x">
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_focus_pen_width">
     <property name="text">
      <string>Focus pen width</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="box_contour_pen_width"/>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="box_focus_pen_width"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
